--- a/libsidplay/unix/my_macros.m4
+++ b/libsidplay/unix/my_macros.m4
@@ -76,12 +76,12 @@

 AC_DEFUN([MY_CHECK_IOS_BIN],
 [
-    AC_MSG_CHECKING([whether standard member ios::binary is available])
+    AC_MSG_CHECKING([whether legacy member ios::bin is available])
     AC_CACHE_VAL(test_cv_have_ios_binary,
     [
         AC_TRY_COMPILE(
-            [#include <fstream.h>],
-            [ifstream myTest(ios::in|ios::binary);],
+            [#include <fstream>],
+            [std::ios::openmode myTest = std::ios::bin;],
             [test_cv_have_ios_binary=yes],
             [test_cv_have_ios_binary=no]
         )
@@ -105,9 +105,9 @@
     AC_CACHE_VAL(test_cv_have_ios_openmode,
     [
         AC_TRY_COMPILE(
-            [#include <fstream.h>
-             #include <iomanip.h>],
-            [ios::openmode myTest = ios::in;],
+            [#include <fstream>
+             #include <iomanip>],
+            [std::ios::openmode myTest = std::ios::in;],
             [test_cv_have_ios_openmode=yes],
             [test_cv_have_ios_openmode=no]
         )
